tasks/main.yml

```diff
@@ -26,5 +26,8 @@
     src: etc/apt/preferences.d/deb_nodesource_com_node.pref.2
     dest: /etc/apt/preferences.d/deb_nodesource_com_node.pref
 
+- name: Uninstall existing installations of Node.js
+  apt: name=nodejs state=absent
+
```

Comment:
Is there some way of making this check it is necessary? It would be way too excessive to have node reinstall itself every time we run through ansible.